Dispatcher/Constants/AppConstants.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  AppConstants.swift
+//  Dispatcher
+//
+//  Created by Ofek Cohen on 30/04/2023.
+//
+
+import Foundation
+
+struct AppConstants {
+    //MARK: Global instanses
+    static let articlesRepository = ArticlesRepository()
+    static let defaults = UserDefaults.standard
+    
+    //MARK: Homepage articles table
+    static let articleCellIdentifier = "articleCell"
+    static let articleCellNibName = "ArticleCell"
+    
+    
+    //MARK: Search
+    static let latestSearchesCellIdentifier = "latestSearchCell"
+    static let latestSearchCellNibName = "LatestSearchCell"
+    static let LATEST_SEARCHES_AMOUNT = 10
+    static let latestSearchesDefualtsKey = "LatestSearches"
+    
+    
+    //MARK: segues
+    static let fromLatestSearchToResults = "fromLatestSearchToResults"
```

Comment:
Let's give the segues a dedicated struct of its own, preferably in a separate file as we could potentially have many of them